openquakeplatform/openquakeplatform/icebox/models.py

```diff
@@ -72,14 +72,15 @@ def process_layers(self):
         # we create a database View per each output, then we publish the
         # db view on geoserver as a postgis vector layer. Finally, we
         # let geonode slurp the geoserver layer
-        for olayer in self.outputlayer_set.all():
```

Comment:
If `self.outputlayer_set.all()` returns `None`, nothing should happen inside the loop, so the additional check looks redundant. What does `self.outputlayer_set.all()` return, motivating that change?